app/src/main/java/org/jellyfin/androidtv/integration/dream/composable/DreamContentLogo.kt

```diff
@@ -18,6 +19,7 @@ import org.jellyfin.androidtv.R
 fun DreamContentLogo() = Box(
 	modifier = Modifier
 		.fillMaxSize()
+		.alpha(0.5f)
```

Comment:
The background for DreamContentLogo is static so the alpha doesn't make any sense here.